doc/rtd/topics/faq.rst

```diff
@@ -151,9 +151,6 @@ provided to the system:
 
     $ cloud-init schema --system --annotate
 
-As launching instances in the cloud can cost money and take a bit longer,
```

Comment:
I removed this since it doesn't make sense where it is. I think this may have previously preceded the section titled "How can I test cloud-init locally before deploying to the cloud?", but I don't think it adds anything there, so I propose deleting it.